package.json

```diff
@@ -1,65 +1,77 @@
 {
   "name": "quilt",
   "version": "1.0.0",
+  "private": true,
   "description": "A loosely related set of packages for JavaScript/TypeScript projects at Shopify",
-  "scripts": {
-    "publish": "lerna publish",
-    "build": "yarn tsc --build packages",
-    "lint": "eslint '**/*.{ts,tsx}'",
-    "lint:changelogs": "node scripts/lint-changelogs.js",
-    "ci:lint-docs": "yarn generate docs && test -z \"$(git status --porcelain)\" || echo 'The root README has not been updated. Run `yarn generate docs` in the root of your quilt directory and try again.'",
-    "_test": "NODE_ICU_DATA=node_modules/full-icu jest",
-    "pretest": "yarn build",
-    "test": "yarn _test --no-coverage",
-    "test:ci": "yarn _test --maxWorkers=3 --coverage",
-    "check": "lerna run check",
-    "release": "yarn lint:changelogs && lerna publish",
-    "release-beta": "lerna publish --dist-tag beta",
-    "clean": "rimraf './packages/*/dist'",
-    "generate": "plop",
-    "generate:package": "plop package && yarn plop docs && prettier --write \"./packages/tsconfig.json\"",
-    "tophat": "tophat"
+  "homepage": "https://github.com/shopify/quilt#readme",
+  "bugs": {
+    "url": "https://github.com/shopify/quilt/issues"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/Shopify/quilt.git"
   },
-  "author": "Shopify Inc.",
   "license": "MIT",
-  "bugs": {
-    "url": "https://github.com/shopify/quilt/issues"
-  },
-  "homepage": "https://github.com/shopify/quilt#readme",
-  "private": true,
+  "author": "Shopify Inc.",
   "workspaces": [
     "packages/*"
   ],
+  "scripts": {
+    "_test": "NODE_ICU_DATA=node_modules/full-icu yarn sk test",
+    "build": "sk build",
+    "check": "lerna run check",
+    "ci:lint-docs": "yarn generate docs && test -z \"$(git status --porcelain)\" || echo 'The root README has not been updated. Run `yarn generate docs` in the root of your quilt directory and try again.'",
+    "clean": "rimraf './packages/*/build' './packages/*/*.d.ts' './packages/*/*.js' './packages/*/*.esnext' './packages/*/*.mjs' '.sewing-kit'",
+    "generate": "plop",
+    "generate:package": "plop package && yarn plop docs && prettier --write \"tsconfig.json\"",
+    "lint": "sk lint",
+    "lint:changelogs": "node scripts/lint-changelogs.js",
+    "publish": "lerna publish",
+    "release": "yarn lint:changelogs && lerna publish",
+    "release-beta": "lerna publish --dist-tag beta",
+    "pretest": "yarn build",
+    "test": "yarn _test",
```

Comment:
I think this may be changing the default `test` command to be in watch mode, which may not be desirable